automatic_host_make.sh

```diff
@@ -0,0 +1,5 @@
+cd SpiNNFrontEndCommon/cpp_common/
+make clean
+make install
+cd ../..
+echo "completed host binary compilation"
```

Comment:
Lines 1–4 should be this:
```
make -C SpiNNFrontEndCommon/cpp_common clean install
```

Reply:
Also doing `set -e` beforehand would be wise; it makes things much less inclined to go wrong silently.